shared/message/Geometry.proto

```diff
@@ -52,50 +46,29 @@ message Ellipse {
  * Four corner points of a quadrilateral
  */
 message Quad {
-
-    /// Top left corner of the quadrilateral
     vec2 tl = 1;
-    /// Top right corner of the quadrilateral
     vec2 tr = 2;
-    /// Bottom left corner of the quadrilateral
     vec2 bl = 3;
-    /// Bottom right corner of the quadrilateral
     vec2 br = 4;
 }
 
-/**
- * Contains the polygon's location
- */
 message Polygon {
-
-    /// {x, y} location of the polygon
     vec2 point = 1;
 }
```

Comment:
I have a feeling that this is meant to be a list of points that are the vertices of a polygon. 
```
message Polygon {
    repeated vec2 point = 1;
}
```

Reply:
That's beyond the scope of this PR. I'll make an issue

Reply:
#426 submitted